packages/entities/entities-plugins/src/components/PluginForm.vue

```diff
@@ -235,15 +235,6 @@ const props = defineProps({
     default: false,
   },
 
-  /**
-   * Support instance names for plugins. This can be removed when KHCP-5872-custom-names-for-plugins is removed.
-   * Enabled by default for KM.
-   */
-  useCustomNamesForPlugin: {
-    type: Boolean,
-    default: false,
-  },
```

Comment:
issue: removing a prop is a breaking change.

Would it be better to deprecate the prop until all usage removal is confirmed in host apps?

If you're actually ready to remove it, you'll need to issue a breaking change commit to bump the major version